docs/platform/one-click/how-to-deploy-cpanel-with-one-click-apps/index.md

```diff
@@ -0,0 +1,76 @@
+---
+author:
+  name: Linode Community
+  email: docs@linode.com
+description: 'cPanel & WHM is a Linux based web hosting control panel and platform that helps you create and manage websites, servers, databases and more with a suite of hosting automation and optimization tools.  Deploy a cPanel & WHM instance using Linode''s One-Click Apps.'
```

Comment:
Two commas for both description and og_description: .."databases, and..." and "..and more, with..." Also, what does the double apostrophe in Linode''s do? We could use quotation marks around the entire description if we're worried about escaping that single apostrophe in "Linode's."